apps/backend/src/modules/friends/friend.repository.ts

```diff
@@ -138,6 +140,24 @@ class FriendRepository implements Repository<UserEntity> {
 		});
 	}
 
+	public async getIsFollowing(
+		currentUserId: number,
+		otherUserId: number,
+	): Promise<boolean> {
+		const userFollowings = await this.userModel
+			.query()
+			.leftJoin(
+				DatabaseTableName.FRIENDS,
+				`${DatabaseTableName.USERS}.id`,
+				"=",
+				`${DatabaseTableName.FRIENDS}.following_id`,
+			)
+			.where(`${DatabaseTableName.FRIENDS}.follower_id`, "=", currentUserId)
+			.where(`${DatabaseTableName.FRIENDS}.following_id`, "=", otherUserId);
+
+		return userFollowings.length > EMPTY_FRIENDS_RESULT;
```

Comment:
```suggestion
		return userFollowings.length > EMPTY_ARRAY_LENGTH;
```
We have it in shared folder